src/UnityExtension/Assets/Editor/GitHub.Unity/UI/BaseWindow.cs

```diff
@@ -54,11 +54,16 @@ public virtual void Finish(bool result)
             RaiseOnClose(result);
         }
 
-        protected void RaiseOnClose(bool result)
+        protected virtual void RaiseOnClose(bool result)
```

Comment:
The override is just calling `ClearOnClose` to clear the callback, and that method is defined here and should always be called after `OnClose` is invoked, so shouldn't we just call it right here instead of making this a virtual and requiring all subclasses to remember to call a magic method?